store/store.go

```diff
@@ -191,10 +212,12 @@ func generateMongoQuery(p *Params) bson.M {
 	//if optional parameters are present, then add them to the query
 	if len(p.Types) > 0 && p.Types[0] != "" {
 		groupDataQuery["type"] = bson.M{"$in": p.Types}
+		skipParamsQuery = true
 	}
 
 	if len(p.SubTypes) > 0 && p.SubTypes[0] != "" {
 		groupDataQuery["subType"] = bson.M{"$in": p.SubTypes}
+		skipParamsQuery = true
```

Comment:
If we specify the type or subType we won't filter on levels ?

Reply:
same as above